sdks/java/core/src/main/java/org/apache/beam/sdk/util/MergingActiveWindowSet.java

```diff
@@ -144,7 +144,7 @@ public void ensureWindowIsActive(W window) {
     checkState(stateAddressWindows != null,
                              "Cannot ensure window %s is active since it is neither ACTIVE nor NEW",
                              window);
-    if (stateAddressWindows.isEmpty()) {
+    if (stateAddressWindows != null && stateAddressWindows.isEmpty()) {
```

Comment:
If I change to use checkNotNull:

```
checkNotNull(timestampMsSinceEpoch,
                    "Cannot interpret value of label %s as timestamp: %s",
                    timestampLabel, value);
```
then findbugs doesn't trigger.   Is that acceptable?  It changes from an IllegalStateException to a NullPointerException that is thrown in the case the field is null.